src/utils/GitHub.js

```diff
@@ -55,6 +56,26 @@ export async function getIssue(repository, issueNumber, accessToken) {
 }
 
 
+/**
+ * @param {object} repository
+ * @param {object} issueNumber
+ * @param {string} title Issue/Note title
+ * @param {string} body Issue/Note body
+ * @param {string} accessToken Github API OAuth access token
+ * @return {object} result
```

Comment:
response from GH, here and throughout... just small reminder to look at GH API docs for the semantics

Reply:
done